### PR TITLE
refactoring + Tmdb api 

### PR DESCRIPTION
**Pull request by andrus92**
**The code has been refactored:**
  1) Improved pop-up windows for movies
  2) Created 2 separate functions to handle click and mouseover events
  3) Added de-registration from events if we change the content of the page

**Implemented fetching the movies from themoviedb.org**

### CODE REVIEW

File: src/tmdbApi.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const API_KEY = '&api_key=c1136013db4178c15f6c05580bac0e31';
```
Comment (SirSerje) [inline src/tmdbApi.js:1]: Use .env file for data like this

File: src/tmdbApi.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const BASE_URL = 'https://api.themoviedb.org/3';
+
+const API_URL = BASE_URL +
```
Comment (SirSerje) [inline src/tmdbApi.js:5]: String literal will look better than concatenation

File: src/index.js [hunk 3]
```diff
@@ -4,59 +4,128 @@ import { sortArrayByName } from './utils';
...
+    movieContainerDiv.addEventListener('mouseover', handleMouseOver);
+}
+
+function unregisterFromMouseOver() {
+    const movieContainerDiv = getMovieContainerDiv();
+    movieContainerDiv.removeEventListener('mouseover', handleMouseOver);
+}
+
+function handleError() {
+    //getMovieContainerDiv().innerText = '⚠️';
```
Comment (SirSerje) [inline src/index.js:88]: Do not commit unused code (stash it locally or make separate branch, if you have a lot of temporary changes)

**Review by SirSerje — CHANGES_REQUESTED** [review]
Do not store private data at public code (use .env file for this), please
So:
1. Put your private api key into .env file
2. Access to variable with process.env.YOUR_VARIABLE_NAME
3. At the request method, make validation like:
If there is process.env.VARIABLE_NAME > make request, otherwise, return mock

**Comment by andrus92** [comment]
These changes have been merged into main branch via https://github.com/andrus92/cinemascope/pull/13.

**Pull request closed without merging**